supercommand.go

```diff
@@ -514,6 +516,13 @@ func (c *SuperCommand) Run(ctx *Context) error {
 	}
 	err := c.action.command.Run(ctx)
 	if err != nil && !IsErrSilent(err) {
+		formatter := DefaultErrorFormatters[c.formattingType]
+		if formatter != nil {
+			logger.Debugf("error stack: \n%v", errors.ErrorStack(err))
+			_ = formatter(ctx.Stderr)
```

Comment:
Make a note, why it's safe for us to ignore this error... is it ever safe to do this?

Reply:
you are right, le me return err